src/test/java/org/pesmypetcare/webservice/builders/PathTest.java

```diff
@@ -203,9 +209,9 @@ public void buildFrequencyOfWashesEntriesCollectionPath() {
     @Test
     public void buildMedicationEntryPath() {
         String medicationName = "painkillers";
-        String path = Path.ofDocument(Collections.medications, userId, petName, date, medicationName);
-        assertEquals("users/" + userId + "/pets/" + petName + "/medications/" + date + "½" + medicationName, path,
-            "Should return the path to the medication entry.");
+        String path = Path.ofDocument(Collections.medications, userId, petName, date + "-" + medicationName);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The String "-" appears 2 times in the file.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5482446)